docs/modeling/processes/variables.md

```diff
@@ -69,10 +69,18 @@ The following is an example payload for `POST rb/v1/process-instances` in the ru
 {
   "payloadType": "StartProcessPayload",
   "processDefinitionId": "Process_MAigMN6p:1:bbfdea14-4907-11ea-9908-deb52b497d16",
-  "variables": {
-    "area_code": 472,
-    "district" : "North"
-   }
+  "variables":[ 
```

Comment:
The format is wrong, it should be:

```
"variables": {
    "area_code": {
      "type": "integer",
      "value": "472"
    },
    "district": {
      "type": "string",
      "value": "North"
    }
  }
```

Actually, both ways are supported, so you can also use the same payload as before:

```
 "variables": {
    "area_code": 472,
    "district" : "North"
   }
```

or you can mix both:
```
"variables": {
    "area_code": {
      "type": "integer",
      "value": "472"
    },
    "district": {
      "type": "string",
      "value": "North"
    },
    "age" : 50
  }
```

This will create `area_code` and `age` as `integer`. When you provide directly the value without type information the engine will try to guess the type.
Important: when you provide the type information the value should always be the string representation of the value. i.e. it should be `"472"` and not `472`.

Reply:
Ah thanks Elias - didn't realise you could do that. I've updated the page :-)